packages/cli/src/ui/components/Footer.tsx

```diff
@@ -139,35 +130,55 @@ export const Footer: React.FC = () => {
             </Text>
           ) : (
             <Text color={theme.status.error}>
-              no sandbox <Text color={theme.text.secondary}>(see /docs)</Text>
+              no sandbox
+              {terminalWidth >= 100 && (
+                <Text color={theme.text.secondary}> (see /docs)</Text>
+              )}
             </Text>
           )}
         </Box>
       )}
 
       {/* Right Section: Gemini Label and Console Summary */}
-      {(!hideModelInfo ||
-        showMemoryUsage ||
-        corgiMode ||
-        (!showErrorDetails && errorCount > 0)) && (
-        <Box alignItems="center" paddingTop={isNarrow ? 1 : 0}>
-          {!hideModelInfo && (
-            <Box alignItems="center">
-              <Text color={theme.text.accent}>
-                {isNarrow ? '' : ' '}
-                {model}{' '}
-                <ContextUsageDisplay
-                  promptTokenCount={promptTokenCount}
-                  model={model}
-                />
+      {!hideModelInfo && (
+        <Box alignItems="center" justifyContent="flex-end">
+          <Box alignItems="center">
+            <Text color={theme.text.accent}>
+              {model}{' '}
+              <ContextUsageDisplay
+                promptTokenCount={promptTokenCount}
+                model={model}
+                terminalWidth={terminalWidth}
+              />
+            </Text>
+            {showMemoryUsage && <MemoryUsageDisplay />}
+          </Box>
+          <Box alignItems="center" paddingLeft={2}>
+            {/* Markdown mode indicator - uncomment to enable
```

Comment:
Fyi @miguelsolorio we probably don't want to enable this in prod but we should figure out a reasonable way to surface that we are in raw markdown display mode. One idea is to look at how we flash Press Ctrl-C again to exit if you press Ctrl+C. We could similarly surface "In raw markdown display mode. Press Ctrl-M again to exit" or similar.

Reply:
Can you work with @miguelsolorio to figure out a solution to this?